<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forceTab</class>
 <widget class="QDialog" name="forceTab">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1  Set force fields</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="aboutTab">
         <property name="toolTip">
          <string>About Tab</string>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bondButton">
         <property name="text">
          <string>bonds</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="typesButton">
         <property name="toolTip">
          <string>Detect atom types.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findCHARMMButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="toolTip">
          <string>Load PRM File</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Save PRM File</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QToolBox" name="defaultForField">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="chargesBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>683</width>
            <height>247</height>
           </rect>
          </property>
          <attribute name="label">
           <string>ATOMS TYPES AND CHARGES</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_33">
           <item row="0" column="0">
          <widget class="QTableWidget" name="chargesTable">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>681</width>
             <height>251</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Atom</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Charge</string>
            </property>
           </column>
          </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="nonbondedBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>683</width>
            <height>247</height>
           </rect>
          </property>
          <attribute name="label">
           <string>NONBONDED</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableWidget" name="nonBondTable">
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>e</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Rmin/2</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="bondBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>683</width>
            <height>247</height>
           </rect>
          </property>
          <attribute name="label">
           <string>BONDS</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="bondsTable">
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Kb</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>b0</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="FBonds">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="anglesBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>683</width>
            <height>247</height>
           </rect>
          </property>
          <attribute name="label">
           <string>ANGLES</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="anglesLayout">
             <item>
              <widget class="QTableWidget" name="anglesTable">
               <column>
                <property name="text">
                 <string>Type</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Ktheta</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>‚≤ê0</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="dihedralsBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>683</width>
            <height>247</height>
           </rect>
          </property>
          <attribute name="label">
           <string>DIHEDRALS</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableWidget" name="dihedralTable">
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Kchi</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>n</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Delta</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
